Makefile

```diff
@@ -23,5 +26,11 @@ setup:
 
 
 .PHONY: test
-test:
+test: build-ts
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --timeout 40000
+
```

Comment:
Nit: Should we replace multiple blank lines in the Makefile with a single blank line?